Yosemite/Yosemite/Stores/Order/ProductInputTransformer.swift

```diff
@@ -137,7 +137,7 @@ public struct ProductInputTransformer {
                                          product: product,
                                          quantity: quantity,
                                          discount: discount ?? defaultDiscount,
-                                         baseSubtotal: baseSubtotal(on: item),
+                                         baseSubtotal: item.pricePreDiscount.decimalValue,
```

Comment:
While not part of this PR, the same as above: How about `netPrice` or `basePrice` for the property? I'd say `baseSubtotal` feels off since both base and subtotal should mean the same price-wise ( a base price without any modifiers).